### ISSUE TITLE
Vanilla Resnet50 Not Computing on iOS via Libtorch (Pytorch Mobile)

### ISSUE THREAD

**Issue by HussainHaris** [pytorch/pytorch#29821]
## 🐛 Bug

Unsure of whether or not Pytorch Mobile iOS supports anything more complex than a resnet34, as I cannot get a vanilla resnet50 to work. I've gotten resnet34 and below as well as mobilenet to work in iOS and call predictions (with the Libtorch library doing the C++ bridging). 

Specifically, in my predictImage function in "TorchModule.mm" in my iOS project this line never computes/moves forward (aka the print statement after is never printed...)

`- (NSInteger)predictImage:(void*)imageBuffer forLabels:(NSInteger)labelCount {
    int outputLabelIndex = -1;
    try {
        std::cout << "\npredictImage";

    at::Tensor tensor = torch::from_blob(imageBuffer, {1, 3, 224, 224}, at::kFloat);
        std::cout << "\npredictImageTwo";
    torch::autograd::AutoGradMode guard(false);
    at::AutoNonVariableTypeMode non_var_type_mode(true);

    auto outputTensor = _impl.forward({tensor}).toTensor();
        std::cout << "\nReceived outputTensor";
`
The "Received outputTensor" is never printed because _impl.forward does not return anything/finish computing for resnet50. This is the issue.

## To Reproduce

This is where I'm getting my resnet50 pt file and the jit trace.
Worked perfectly fine for resnet34, resnet18, and mobilenet. Also yes I am adding a softmax at the end to normalize the values between 0 and 1 but that shouldn't cause an issue and didn't with the latter models.

**This is Python code for tracing the model**
(Sorry for some reason it won't format but it's pretty simple)

device = torch.device('cpu')

model = models.resnet50(pretrained=True)
model.load_state_dict(models.resnet50(pretrained=True).state_dict())
model = nn.Sequential(
    #ImageScale(),
    model,
    nn.Softmax(1)
)
model.eval()
input_tensor = torch.rand(1,3,224,224)
script_model = torch.jit.trace(model, input_tensor)
script_model.save("models/resnet50.pt")

## Expected behavior
I expect the resnet50 to not freeze/return the output tensor in iOS via Pytorch mobile.

Some useful tags for this would be Libtorch, mobile, jit, iOS

Thanks in advance!

**Comment by HussainHaris** [comment]
Code is much easier to view here: 
https://discuss.pytorch.org/t/pytorch-mobile-ios-resnet50-not-computing/61014

**Comment by xta0** [comment]
Hi Hussain, 

As we've discussed in the PyTorch forum, this is a known issue, and we've fixed it - #29885  .If you want to try out the fix, you need to pull the latest code from master and re-compile the static libraries from source code. Note that the fix will be available in the release of Cocoapods.

**Comment by xta0** [comment]
@HussainHaris 
I'm going to close this issue. The fix has been landed. Let me know if you have any questions in the forum thread.
